clients/vscode/src/chat/utils.ts

```diff
@@ -221,3 +222,49 @@ export function generateLocalNotebookCellUri(notebook: Uri, handle: number): Uri
   const fragment = `${p}${s}s${Buffer.from(notebook.scheme).toString("base64")}`;
   return notebook.with({ scheme: DocumentSchemes.vscodeNotebookCell, fragment });
 }
+
+export function uriToListFileItem(uri: Uri, gitProvider: GitProvider): ListFileItem {
+  const filepath = localUriToChatPanelFilepath(uri, gitProvider);
+  let label: string;
+
+  if (filepath.kind === "git") {
+    label = filepath.filepath;
+  } else {
+    const workspaceFolder = workspace.getWorkspaceFolder(uri);
+    if (workspaceFolder) {
+      label = path.relative(workspaceFolder.uri.fsPath, uri.fsPath);
+    } else {
+      label = path.basename(uri.fsPath);
+    }
+  }
```

Comment:
Please try to use method `workspace.asRelativePath`:
```suggestion
  const label = worksapce.asRelativePath(uri);
```